src/ui/actions_main_area.ui

```diff
@@ -64,20 +64,30 @@
        </widget>
       </item>
       <item>
-       <widget class="QLineEdit" name="lineEdit_action_name"/>
+       <widget class="QLineEdit" name="lineEdit_action_name">
+        <property name="toolTip">
+         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This text should have been replaced by code in dlgTriggerEditor::slot_action_selected()!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
It was an internal reminder whilst using the `QDesigner` plugin to the Qt IDE that tooltip code to do just that __was needed__, and it could not be left to a static value set in the .ui file.  If I hadn't of done the needed other step it would show up as a reminder! :frowning_face::hammer: